browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -512,6 +512,12 @@ This file contains all "about" strings.  It is set to NOT be translated, in tran
       <message name="IDS_FILTER_LISTS_TITLE" desc="Title for Filter Lists settings.">
         Filter Lists
       </message>
+      <message name="IDS_DEFAULT_FILTER_LISTS" desc="Title for Default filter lists.">
+        Default filter lists
+      </message>
+      <message name="IDS_DEFAULT_FILTER_LISTS_SUMMARY" desc="Summary text for Default filter lists.">
+        Additional popular community lists. Note that enabling too many filters will degrade browsing speeds.
+      </message>
```

Comment:
I'm concerned about calling these "default" filter lists, since they are generally _off_ by default. We already reserve that naming for the _on_-by-default lists (although we don't show it in the UI anywhere).

Historically we've called these ones "regional lists". Ever since `brave://adblock` was replaced by `brave://settings/shields/filters` on Desktop, they're just labeled "Filter lists".

My recommendation is to follow the naming on Desktop (`Filter lists`). The title of the overall settings page can also be renamed to match accordingly (`Content Filters`).

Reply:
@timchilds @deeppandya what do you think? Should we follow the desktop or ios?

Reply:
I agree we should remove the default and just keep the filter list as a copy.